admiral/pkg/clusters/DRUtil.go

```diff
@@ -0,0 +1,47 @@
+package clusters
+
+import (
+	"context"
+	"github.com/istio-ecosystem/admiral/admiral/pkg/controller/common"
+	log "github.com/sirupsen/logrus"
+	"strings"
+)
+const  READ_WRITE_ENABLED = false
```

Comment:
just a nitpick, can we make this camelcase?

Reply:
This is fixed